pkg/store/prometheus.go

```diff
@@ -699,3 +720,55 @@ func (p *PrometheusStore) LabelSet() []labelpb.ZLabelSet {
 func (p *PrometheusStore) Timestamps() (mint int64, maxt int64) {
 	return p.timestamps()
 }
+
+func (p *PrometheusStore) getMatchedSeriesCount(matchers []*labels.Matcher, start, end int64) (int, error) {
```

Comment:
Maybe this could be moved to `pkg/promclient/promclient.go` and then we could use function `formatTime` instead of creating `timeToPromTimestamp`?